### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/526491af3ef6e284deb45aa7433c3be98f818cf7

### CODE REVIEW

**Comment by flodolo** [comment]
This is exposing 4 pages, 3 of which haven't been touched in 2 years.

That content needs to be reviewed before merging this.

**Comment by flodolo** [comment]
@peiying2 
Not sure I understand your last comment.

This PR is basically exposing 4 pages for translation
https://github.com/mozilla/bedrock/tree/main/l10n/en/products/vpn/more

The corresponding pages in www-l10n haven't been touched in 2 years, and I'm sure they contain some of the problems that we've been trying to fix over the last months (abuse of terms, use of variables for anchors URLs).
https://github.com/mozilla-l10n/www-l10n/tree/master/en/products/vpn/more

**Comment by peiying2** [comment]
@nathan-barrett This PR would enable localization for all the locales. Three of the four files were localized two years ago and one was as late last year. There are two things that I'd like to bring to your attention:
- The re-enabled files contain content that's out of date. They need to be reviewed and updated
- It appears that the changes are the result of URL redirects to the resource center. The content there (through Contentful) is only localized in 10 languages: de, fr, es-ES, it, ja, nl, pl, pt-BR, ru, zh-CN. In other words, this PR will open up all the locales on mozilla.org but only 10 locales will see the content on Resource Center in their languages.

File: en/products/vpn/landing.ftl
Comment (peiying2) [inline en/products/vpn/landing.ftl:116]: For third party brand names, we stop putting them in placeholders, so just spell it like normal text. 

For the reference tag, `<a href="{ $url }">`, we suggest to use `<a { $url }>` as the extra codes often generates extra errors in localized strings.

File: en/products/vpn/landing.ftl
Comment (peiying2) [inline en/products/vpn/landing.ftl:121]: same as above

**Review by peiying2 — CHANGES_REQUESTED** [review]
For third party brand names, we stop putting them in placeholders, so just spell it like normal text. 

For the reference tag, `<a href="{ $url }">`, we suggest to use `<a { $url }>` as the extra codes often generates extra errors in localized strings.

File: en/products/vpn/landing.ftl
Comment (alexgibson) [reply en/products/vpn/landing.ftl:121]: This is an old string so we can't change it (it just moved a line in the diff)

**Comment by alexgibson** [comment]
@peiying2 @flodolo I think Nathan is out until next week, but it looks like these other changes are reactivating some older pages. I understand your wish to remove things like third party brand names and simplifying the URLs, but it looks like updating that old content would require quite a lot of old strings needing re-translation?

**Comment by flodolo** [comment]
> but it looks like updating that old content would require quite a lot of old strings needing re-translation?

They're only translated by vendors for the limited set of locales, they're not translated by all the other locales (because they were not in the TOML).

I didn't go through the content in depth, but I couldn't help noticing how obsolete it is in some parts (e.g. references to FPN?).

**Comment by alexgibson** [comment]
It looks like https://github.com/mozilla/bedrock/pull/12938/files removed pages that were translated in pontoon, not by vendor. I think that's what's being reinstated here (?), but I'll let @nathan-barrett comment when he's back as to the current state of content.

**Comment by flodolo** [comment]
> removed pages that were translated in pontoon, not by vendor. I think that's what's being reinstated here

That's one part of the puzzle I was missing. 

Obviously, luck wants that we [removed](https://github.com/mozilla-l10n/www-l10n/pull/358) those files from all locales 2 weeks ago, since they were not exposed in Pontoon 🤦🏼  

Then the next steps here is to:
* Keep the content unchanged.
* Recover those files from the previous version.

I'll look into this next week.

**Review by peiying2 — CHANGES_REQUESTED** [review]
- Need to add new file entries to the .toml file: features, landing-2023, pricing-2023. 
- Yes, we can ignore the change request for landing.ftl as it's going to be obsolete.

File: en/products/vpn/features.ftl
Comment (flodolo) [inline en/products/vpn/features.ftl:24]:
```suggestion
vpn-features-connect-up-to = { $devices -> 
    [one] Connect up to { $devices } device
   *[other] Connect up to { $devices } devices
}
```

We need a plural for this type of strings

File: en/products/vpn/features.ftl
Comment (flodolo) [inline en/products/vpn/features.ftl:19]:
```suggestion
vpn-features-more-than = { $servers -> 
    [one] More than { $servers } server in { $countries }+ countries
   *[other] More than { $servers } servers in { $countries }+ countries
}
```

Sadly, 2 plurals in a string is very complicated in our current Fluent implementation.

@peiying2 
I wonder how this is going to show up in Smartling. We need extra attention from translators.

File: en/products/vpn/features.ftl
Comment (flodolo) [inline en/products/vpn/features.ftl:25]:
```suggestion
vpn-features-supported-platforms = Supported on Windows, macOS, Android, iOS, and Linux operating systems.
```

Do we use Oxford comma in mozorg content? In case, there are a couple more to fix.

File: en/products/vpn/features.ftl
Comment (flodolo) [inline en/products/vpn/features.ftl:76]:
```suggestion
vpn-features-made-with-open-source-code = { -brand-name-mozilla-vpn } is made with open-source code, meaning that all the code is publicly accessible. See our <a { $github }>GitHub repository</a>.
```

File: en/products/vpn/landing-2023.ftl
Comment (flodolo) [inline en/products/vpn/landing-2023.ftl:23]:
```suggestion
vpn-landing-log-into-your-bank-or = Log in to your bank or doctor’s office from the airport, cafe or anywhere, with peace of mind.
```
https://acorn.firefox.com/latest/content/word-list/g-m-NIU35GX3

File: en/products/vpn/landing-2023.ftl
Comment (flodolo) [inline en/products/vpn/landing-2023.ftl:32]: See plural suggestion in the other file (same for the string after).

File: en/products/vpn/pricing-2023.ftl
Comment (flodolo) [inline en/products/vpn/pricing-2023.ftl:18]: See plural suggestion in the other file (same for the string after).

File: en/products/vpn/pricing-2023.ftl
Comment (flodolo) [inline en/products/vpn/pricing-2023.ftl:35]:
```suggestion
vpn-pricing-if-you-purchased = If you purchased your subscription through in-app purchase from the Apple App Store or the Google Play Store, your payment is subject to the terms and conditions of the store. You must direct any billing and refund inquiries for such purchases to Apple or Google, as appropriate.
```

This makes it sound like Google's depends on Apple's App Store terms…

**Comment by flodolo** [comment]
> I'll look into this next week.

Files have been restored in #364

**Comment by nathan-barrett** [comment]
Sorry for being out for most of this conversation - but this was a complicated PR that I can provide a little more context: 

The pages located at `vpn/more/*` were removed earlier this year, with the thought that the pages in the VPN resource center would be able to replace them. 

We realized that since the pages in the resource center are translated with Smartling the set of locales translated are more limited to what is translated in Pontoon. Back in August when this work started, I noticed that the translation files for the original pages were still in the `www-l10n` repo, so I thought (probably mistakenly) that we could bring the pages at `vpn/more/*` without any issues.

Regarding the redirects between the `more/*` pages and the resource center, I have this logic in place:

- When a `/more/*` url is requested the user should be forwarded to the corresponding  /`resource-center` url
- If the rc article is not available in their requested locale bedrock should display the /more/ article if it is available in their language.
- If neither is available in their language bedrock should forward to the English rc article.

File: en/products/vpn/landing-2023.ftl
Comment (craigcook) [reply en/products/vpn/landing-2023.ftl:27]: It means streaming media from other countries that may be region-locked. For example there's a BBC app that lets you watch BBC programs but only if you're in the UK (or using a VPN with a UK-based server). We may be able to rephrase this to be more clear.

File: en/products/vpn/features.ftl
Comment (craigcook) [reply en/products/vpn/features.ftl:25]: I think the style guide says no Oxford comma.

File: en/products/vpn/features.ftl
Comment (craigcook) [reply en/products/vpn/features.ftl:19]: For this string as well as the number of devices string above, we know these will ALWAYS be numbers greater than 1. Do we still need a singular form as a matter of best practice (when number is a variable), or might these be an exception?

Comment (flodolo) [reply en/products/vpn/features.ftl:19]: > we know these will ALWAYS be numbers greater than 1

This approach is valid for "simple" languages (English, romance languages, German), but not for a lot of other languages.

For a language like [Russian](https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html#ru), 21 needs a different noun than 20, 22-24, or 25-29. Without knowing the number in advance, there's no way to translate it correctly (the alternative is to disconnect the number from the noun, e.g. putting it between parentheses). 

In general, when there is a variable number with a noun, [we need a plural form](https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html).

**Review by peiying2 — APPROVED** [review]

**Pull request merged**
